plugins/module_utils/_api_data.py

```diff
@@ -844,6 +844,19 @@ def join_path(path):
             )),
         ],
     ),
+    ('routing', 'filter', 'num-list'): APIData(
+        versioned=[
+            ('7', '>=', VersionedAPIData(
```

Comment:
Since this is only supported since RouterOS 7.1 (according to the changelog fragment you added), this should be:
```suggestion
            ('7.1', '>=', VersionedAPIData(
```

Reply:
As far as I know, this is there from the beginning, and the first version is 7.1  (according to the changelogs of MikroTik). So yeah :) But the same holds for the other functions.

I you want, I could change the changelog to `7` instead of `7.1`.

Reply:
Maybe change the changelog to 7 then.